src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -909,6 +909,18 @@ public BuildResult BuildRequest(BuildRequestData requestData)
         /// </remarks>
         /// <exception cref="InvalidOperationException">Thrown if there is no build in progress.</exception>
         public void EndBuild()
+        {
+            EndBuild(false);
+        }
+
+        /// <summary>
+        /// Signals that no more build requests are expected (or allowed) and the BuildManager may clean up.
+        /// </summary>
+        /// <remarks>
+        /// This call blocks until all currently pending requests are complete.
+        /// </remarks>
+        /// <exception cref="InvalidOperationException">Thrown if there is no build in progress.</exception>
+        public void EndBuild(bool skipLoggingBuildFinished)
```

Comment:
This is to prevent "Build Succeeded 0 Warnings 0 Errors" after you finish a build if you enable the feature, as we should then not log anything but the json (or other format).